test/Sentry.AspNetCore.Tests/SentryTracingMiddlewareTests.cs

```diff
@@ -169,11 +163,12 @@ public async Task Transaction_is_started_automatically_from_incoming_trace_heade
             await client.SendAsync(request);
 
             // Assert
-            transaction.Should().NotBeNull();
-            transaction?.Name.Should().Be("GET /person/{id}");
-            transaction.TraceId.Should().Be(SentryId.Parse("75302ac48a024bde9a3b3734a82e36c8"));
-            transaction.ParentSpanId.Should().Be(SpanId.Parse("1000000000000000"));
-            transaction.IsSampled.Should().BeFalse();
+            sentryClient.Received(1).CaptureTransaction(Arg.Is<Transaction>(t =>
+                t.Name == "GET /person/{id}" &&
```

Comment:
The issue was `trasnaction?.` and transaction was null so assert wasn't being called?

Reply:
No, it was fine. The problem is that the transaction was created correctly, it was not sent correctly. So I widened the test scope to cover that (by testing against `ISentryClient`).